x-pack/plugins/lens/public/indexpattern_datasource/operations/operations.ts

```diff
@@ -146,8 +146,8 @@ export function getAvailableOperationsByMetadata(indexPattern: IndexPattern) {
 
   operationDefinitions.sort(getSortScoreByPriority).forEach((operationDefinition) => {
     if (operationDefinition.input === 'field') {
-      indexPattern.fields.forEach((field) => {
```

Comment:
I don't see how these changes improve the code - it's still nested to the same depth (plus `addToMap` does not actually return something). Can you elaborate?

Reply:
Not a particular speed up here.
I did some small profiling and numbers are bit lower than with all the conditionals: seeing no harm in particular change I kept it, but it can easily reversed.

Reply:
> with all the conditionals

Which conditionals do you mean? I wasn't aware this was a performance optimization, can you explain how it could speed things up? I think I'm missing something here

Reply:
I meant "from the previous implementations with the `if/else` conditionals".
It may be an anecdotal to my machine, so no strong opinion on these.